anchor/anchorRecord.go

```diff
@@ -207,11 +242,30 @@ func UnmarshalAndValidateAnchorEntryAnyVersion(entry interfaces.IEBEntry, public
 	return ar, valid, err
 }
 
+// CreateAnchorRecordFromDBlock creates new AnchorRecord from dBlock
 func CreateAnchorRecordFromDBlock(dBlock interfaces.IDirectoryBlock) *AnchorRecord {
```

Comment:
This function isn't used anywhere, I'd just delete it and all its references in the tests

Reply:
I deleted CreateAnchorRecordFromDBlock() and removed it from the files it was used, just duplicating a small amount of code in their respective locations